pom.xml

```diff
@@ -29,7 +29,7 @@
 
 	<groupId>com.ixortalk</groupId>
 	<artifactId>ixortalk-oauth2-spring-boot-starter</artifactId>
-	<version>1.0.11-SNAPSHOT</version>
+	<version>1.0.11</version>
```

Comment:
Should remain `SNAPSHOT` version.  Might be interesting to bump the minor version since this change contains breaking changes.